### PR TITLE
ranking.goo.ne.jp

### LINKED ISSUE #134169: ranking.goo.ne.jp
### Issue URL (Annoyance)
 [https://ranking.goo.ne.jp/column/s4805](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Franking.goo.ne.jp%2Fcolumn%2Fs4805)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.1.37
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

Fix #134169

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -44,6 +44,7 @@ iine-kunitachi.net,2d-erocafe.com,acueducto.jp,akashi-journal.com,analsexgay.net
 !
 !
 makitani.net##.ablock
+||image1.rank-king.jp/item/image/original/8408712.jpg
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:47]: The image path is not likely to be stable, better `ranking.goo.ne.jp###sticky_adsense`

File: MobileFilter/sections/specific_web.txt [hunk 2]
```diff
@@ -53,6 +53,7 @@ berkem.ru,pixiv.net,cartoonpornvideos.com,m.economictimes.com,beethovenmaster.co
 rakuraku-info.jp,sugoren.com,dailynewsonline.jp,yakuji.co.jp,asahi.com,m.winporn.com,slate.com,sankei.com,gameranbu.jp,seiga.nicovideo.jp,m.arukikata.co.jp,setn.com,hateblo.jp,hatenablog.com,hatenablog.jp,hatenadiary.com,hatenadiary.jp,hatenadiary.org,syosetu.org##.ad
 !
 vtuber-matome.com##.AD-frame
+||image1.rank-king.jp/item/image/original/8408327.jpg
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:56]: Same, `ranking.goo.ne.jp##.banner`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt]: ok

File: MobileFilter/sections/specific_web.txt (hunk 2 above)
Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt]: ok

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**